Tests/package.lisp

```diff
@@ -1,14 +1,8 @@
 (cl:defpackage #:clim-tests
-  (:use
-   #:clim-lisp
-   #:clim
-   #:clime
-   #:fiveam)
-  (:import-from
-   #:climi
-   #:coordinate=)
-  (:export
-   #:run-tests))
+  (:use #:clim-lisp #:clim #:clime #:fiveam)
+  (:shadowing-import-from #:fiveam #:test)
```

Comment:
How about selectively importing from `fiveam` instead? We only use a few symbols and issues like this would be less likely in the future.

Reply:
Discussed in IRC.